### PR TITLE
Oder product

### PR DESCRIPTION
**Pull request by tranquanlam**

### CODE REVIEW

**Comment by tranquanlam** [comment]
ready

File: src/store/reducer/product-reducer.js [hunk 1]
```diff
@@ -1,49 +1,36 @@
...
-        .catch(function (error) {
-            console.log(error);
-        });
-        console.log("axios");
-        hostPro.forEach(element => {
-        console.log(element)
-            });
-        state = {...state,hotproduct : data}
-    return state
-}
```
Comment (phamvanbk11) [inline src/store/reducer/product-reducer.js]: Sao xóa hết vậy

**Comment by phamvanbk11** [comment]
commented

**Comment by tranquanlam** [comment]
ready

**Pull request merged**